scripts/increment_auto_off.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+# A program that ticks a counter down until deactivating log_persistence
+
+import logging
+import sys
+import json
+import time
+
+import requests
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+sh = logging.StreamHandler(sys.stdout)
+logger.addHandler(sh)
+
+loop = True
+
+while loop:
+  try:
+    response = requests.get('http://localhost:5001/settings/persist_logs', timeout=10)
+    state = response.json()
+
+    if state["persist_logs"]:
+      future_persist_state = state["auto_off_delay"] != 1
+      delay = state["auto_off_delay"] - 1
```

Comment:
you _need_ to check for these keys existence. please add this file to our `mypy` checking and resolve all issues it reports.

Reply:
![image](https://github.com/user-attachments/assets/bd938469-5baf-465c-911c-18b2cc0bb735)

I feel like I've added them to our workflows properly, though picking through the completed workflows I don't see the file getting checked either way. Here's something that approximates a badge of approval from the checkers, though.